FlowCryptCommon/Extensions/UIViewExtensions.swift

```diff
@@ -8,52 +8,11 @@ public extension UIView {
     func constrainToEdges(_ subview: UIView, insets: UIEdgeInsets = .zero) {
         subview.translatesAutoresizingMaskIntoConstraints = false
 
-        let topConstraint = NSLayoutConstraint(
-            item: subview,
-            attribute: .top,
-            relatedBy: .equal,
-            toItem: self,
-            attribute: .top,
-            multiplier: 1.0,
-            constant: insets.top
-        )
-
-        let bottomConstraint = NSLayoutConstraint(
-            item: subview,
-            attribute: .bottom,
-            relatedBy: .equal,
-            toItem: self,
-            attribute: .bottom,
-            multiplier: 1.0,
-            constant: -insets.bottom
-        )
-
-        let leadingConstraint = NSLayoutConstraint(
-            item: subview,
-            attribute: .leading,
-            relatedBy: .equal,
-            toItem: self,
-            attribute: .leading,
-            multiplier: 1.0,
-            constant: insets.left
-        )
-
-        let trailingConstraint = NSLayoutConstraint(
-            item: subview,
-            attribute: .trailing,
-            relatedBy: .equal,
-            toItem: self,
-            attribute: .trailing,
-            multiplier: 1.0,
-            constant: -insets.right
-        )
-
-        addConstraints([
-            topConstraint,
-            bottomConstraint,
-            leadingConstraint,
-            trailingConstraint
-        ])
+        let guide = self.safeAreaLayoutGuide
+        subview.trailingAnchor.constraint(equalTo: guide.trailingAnchor, constant: -insets.right).isActive = true
+        subview.leadingAnchor.constraint(equalTo: guide.leadingAnchor, constant: insets.left).isActive = true
+        subview.topAnchor.constraint(equalTo: guide.topAnchor, constant: insets.top).isActive = true
+        subview.bottomAnchor.constraint(equalTo: guide.bottomAnchor, constant: -insets.bottom).isActive = true
```

Comment:
it seems we don't need additional inset on the bottom, as it produces additional empty space on the bottom. and also bottom corners have some strange border radius:

<img width="346" alt="Screenshot 2023-12-05 at 10 29 51" src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/3ba0cd55-8564-4a9a-acd8-12e7162debbd">

Reply:
Nice catch. Just fixed